deploy/examples/create-external-cluster-resources.py

```diff
@@ -1000,6 +1000,8 @@ def validate_rgw_endpoint(self):
     def _gen_output_map(self):
         if self.out_map:
             return
+        if self._arg_parser.rgw_endpoint:
+            self._arg_parser.rgw_endpoint = self.convert_fqdn_rgw_endpoint_to_ip(self._arg_parser.rgw_endpoint)
```

Comment:
I wonder how this fails if the FQDN is not revolvable or if the IP is empty. Can you help?